lib/map/config.js

```diff
@@ -0,0 +1,7 @@
+var CONSTANTS = require('../constants');
```

Comment:
Improper explanation. I meant that this file is not required anywhere so it would not be loaded.